src/sentry/static/sentry/app/views/organizationEventsV2/eventModalContent.jsx

```diff
@@ -49,7 +51,11 @@ const EventModalContent = props => {
           })}
       </HeaderBox>
       <ContentColumn>
-        <EventInterfaces event={event} projectId={projectId} />
+        {get(event, 'type') === 'transaction' ? (
```

Comment:
You don't need `get()` here. `event` will always have a type property.

Reply:
thanks. I assumed it was optional from the proptype in SentryTypes.

I'll change this.